src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/PlatformInterop.java

```diff
@@ -353,4 +356,23 @@ public static void loadImageFromFont(Context context, @ColorInt int color, Strin
             .override(Target.SIZE_ORIGINAL, Target.SIZE_ORIGINAL)
             .into(target);
     }
+
+    public static final ColorStateList getColorStateListForStyleableAttribute(Context context, String styleable, int resId, int index) {
+        try {
+            Field[] fields = R.styleable.class.getFields();
+            Field field = R.styleable.class.getField(styleable);
+            int [] attributes = (int[]) field.get(R.styleable.class);
```

Comment:
Was this using C# System.Reflection before? Is there a way to do this and not use Java reflection? Use the field directly?

Reply:
The only downside is that this method could only be used for the `Toolbar` if we did that. If we pass in the "string" name of the field, we can then use it in various places for different fields. 

I guess its a trade off. We can be specific or write a generic method. The latter will need reflection, but the former as you said, will be hardcoded to only work for one field.

Reply:
Yes I’d just hardcode for one field, especially if this is used in one place. I’d we avoid the reflection, it will help startup.